### ISSUE TITLE
Unable to use with Nuxt.js

### ISSUE THREAD

**Issue by andylow** [antoniandre/vueper-slides#6]
Cause "ReferenceError: document is not defined" when use with Nuxt.js

**Comment by antoniandre** [comment]
This is not due to Vueper Slides, as mentioned here https://nuxtjs.org/faq/window-document-undefined, the components created and before create hooks are also triggered from server-side where no document / window is available.

Since Nuxt 1.0.0 RC7, you can use the `<no-ssr>` tag to allow a component on client-side only.

      <no-ssr>
        <vueper-slides>
          <vueper-slide v-for="i in 5" :key="i" :title="i.toString()"></vueper-slide>
        </vueper-slides>
      </no-ssr>

This will do.

then in the script part, include Vueper Slides like so:

```
import { VueperSlides, VueperSlide } from 'vueperslides'
import 'vueperslides/dist/vueperslides.min.css'

export default {
  components: { VueperSlides, VueperSlide }
}

```

**Comment by antoniandre** [comment]
SSR is now supported in Vueper Slides since version 1.20.3. Now Vueper Slides v2 is released and tested on Nuxt 2: #55
